ams/optimizer/common/src/main/java/com/netease/arctic/optimizer/common/OptimizerExecutor.java

```diff
@@ -26,30 +26,48 @@
 import com.netease.arctic.optimizing.TableOptimizing;
 import com.netease.arctic.utils.ExceptionUtil;
 import com.netease.arctic.utils.SerializationUtil;
+import org.apache.commons.lang3.StringUtils;
 import org.apache.iceberg.common.DynConstructors;
 import org.apache.thrift.TException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.nio.ByteBuffer;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Consumer;
 
 public class OptimizerExecutor extends AbstractOptimizerOperator {
 
   private static final Logger LOG = LoggerFactory.getLogger(OptimizerExecutor.class);
 
   private final int threadId;
+  private Map<String, String> runtimeContext = new ConcurrentHashMap<String, String>();
```

Comment:
I'm not sure if placing this logic in the `OptimizerExecutor` is a good approach; it seems that other types of optimizers currently don't require this logic. Can we consider migrating them to the `FlinkExecutor` instead?

Reply:
I agree that the relevant logic of flink is only effective for flink.

Reply:
done